libraries/ms-helper/inc/exported_enums.h

```diff
@@ -115,3 +115,16 @@ typedef enum {
   EE_POWER_DISTRIBUTION_FAULT_REASON_SEQUENCE_RETRY_EXPIRY,
   NUM_EE_POWER_DISTRIBUTION_FAULT_REASONS,
 } EEPowerDistributionFaultReason;
+
+// Events raised by Driver Controls: Center Console
+typedef enum {
+  EE_CENTER_CONSOLE_DIGITAL_INPUT_BPS = 0u,
```

Comment:
is there a specific reason for this to be `0u` as opposed to just `0`?

Reply:
Just a habit from me working with MISRA C :upside_down_face: